lib/membrane_ice_plugin/ice_endpoint.ex

```diff
@@ -161,13 +161,13 @@ defmodule Membrane.ICE.Endpoint do
 
   def_input_pad :input,
     availability: :on_request,
-    caps: :any,
+    accepted_format: _any,
     mode: :pull,
     demand_unit: :buffers
 
   def_output_pad :output,
     availability: :on_request,
-    caps: {RemoteStream, content_format: nil, type: :packetized},
+    accepted_format: {RemoteStream, content_format: nil, type: :packetized},
```

Comment:
```suggestion
    accepted_format: %RemoteStream{content_format: nil, type: :packetized},
```
Take a look at `Update pads definitions` section in https://github.com/membraneframework/membrane_core/blob/master/guides/upgrading/v0.11.md

Reply:
This is still undone